src/app/app.component.ts

```diff
@@ -54,6 +54,9 @@ export class AppComponent {
   }
 
   changeLocale(locale: string) {
+    if (locale === this.locale) {
```

Comment:
can be removed also, since sbb-select doens't trigger this when you select the same item than the one already selected